From 93787153017a0b83e1cbbc88ee785778e68f9d68 Mon Sep 17 00:00:00 2001
From: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
Date: Thu, 31 Mar 2016 15:16:10 +0530
Subject: [PATCH 0206/1852] media: xilinx: Fix "maybe used uninitialized"
 warning

commit ad8ad7c19eb2e5783ade8f0a1a892cd06c3a16ec from
https://github.com/Xilinx/linux-xlnx.git

Initialize the 'fmt' variable to keep the compiler quiet.
There is no "used uninitialized" in this case.

Signed-off-by: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-tpg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-tpg.c b/drivers/media/platform/xilinx/xilinx-tpg.c
index c97c38986fc8..498cd0c396f9 100644
--- a/drivers/media/platform/xilinx/xilinx-tpg.c
+++ b/drivers/media/platform/xilinx/xilinx-tpg.c
@@ -231,7 +231,7 @@ static int xtpg_s_stream(struct v4l2_subdev *subdev, int enable)
 	}
 
 	if (xtpg->is_hls) {
-		u32 fmt;
+		u32 fmt = 0;
 
 		switch (xtpg->vip_format->code) {
 		case MEDIA_BUS_FMT_UYVY8_1X16:
-- 
2.31.1

